src/tools/genpolicy/rules.rego

```diff
@@ -545,6 +545,29 @@ allow_env(policy_process, input_process, sandbox_name) {
     print("allow_env: success")
 }
 
+# Allow input env variables that match with request_defaults.
+allow_env_var(policy_process, input_process, env_var, sandbox_name) {
+    print("allow_env_var regex 1: some allow_env_regex match env_var")
+
+    some policy_var_regex in policy_data.request_defaults.CreateContainerRequest.allow_env_regex
+    regex.match(policy_var_regex, env_var)
+
+    print("allow_env_var regex 1: success")
+}
+
+# Allow input env variables that match with request_defaults, after substituting $(sandbox-name).
+allow_env_var(policy_process, input_process, env_var, sandbox_name) {
+    print("allow_env_var regex 2: some allow_env_regex match env_var")
+
+    some policy_var_regex in policy_data.request_defaults.CreateContainerRequest.allow_env_regex
+    policy_regex = replace(policy_var_regex, "$(sandbox-name)", sandbox_name)
```

Comment:
Do we need this second set of allow rules? 

I don't see any $(sandbox-name) in the current allow_env_regex - do you expect some of them in the future?